pipeline_dp/combiners.py

```diff
@@ -64,6 +64,10 @@ def compute_metrics(self, accumulator):
     def metrics_names(self) -> List[str]:
         """Return the list of names of the metrics this combiner computes"""
 
+    @abc.abstractmethod
+    def explain_computation(self) -> str:
```

Comment:
Can we add a doc for what this method is supposed to do and how it should look, e.g. is this supposed to be an explanation understandable by PipelineDP users?